src/constants/menus.ts

```diff
@@ -25,7 +25,7 @@ export type MenuItem<MenuItemValue, MenuItemTypes = string> = {
   labelRight?: React.ReactNode;
   tag?: React.ReactNode;
   slotAfter?: React.ReactNode;
-  description?: string;
+  description?: React.ReactNode;
```

Comment:
We were passing stringGetter() results which contain react nodes before but casting them to string to make it work. So we know it actually works because it's been doing the right thing at runtime the whole time. This type change is just making it more accurate to our usage.